### DISCUSSION TITLE
Im student and GitHub just doesn't aceept

### DISCUSSION THREAD

**Discussion by mintrix** [community/community#60119] [GitHub Education]
### Hi everyone,

hi everyone.

well, I tried in 2018 to join in the student starter pack, but they rejected me, now I'm trying again but I doesn't have an email from my university and my master, got system says that I need to have from both of them, but I don't have, so I can't even moving forward to sent any proof, I trying to send email, I just don't know what to do anymore. Can someone give me some light here?

**Comment by samus-aran** [comment]
Hi @mintrix 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been associated with this school.

We would recommend to you speak to your institution regarding to acquiring an email address to proceed.

**Comment by mintrix** [comment]
Hi Ashley!! Thank you very much for answering me

the problem it's that the both institutions that I'm student doesn't give me any email on the way GitHub is requiring, but I don't know why, both of them GitHub have the institution to select and I can't moving forward because of this system, I can send proofs of payments, test scores, or something like that, but I don't know how to send it, I know that I can do this bur I don't know how to proceed

I have to do my final project and I would like to use the GitHub copilot for it

thank you very much again

Enviado do Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by nonvenomous** [reply]
Same issue for me since a month without any solution or response back from support.
Big thread over here https://github.com/orgs/community/discussions/59360
